exporters/stats/stackdriver/src/main/java/io/opencensus/exporter/stats/stackdriver/StackdriverExportUtils.java

```diff
@@ -425,6 +434,143 @@ static void setResourceForBuilder(
+    final List<io.opencensus.metrics.export.TimeSeries> percentileTimeSeries = new ArrayList<>();
+    final List<io.opencensus.metrics.export.TimeSeries> summaryCountTimeSeries = new ArrayList<>();
+    final List<io.opencensus.metrics.export.TimeSeries> summarySumTimeSeries = new ArrayList<>();
+    for (final io.opencensus.metrics.export.TimeSeries timeSeries :
+        summaryMetric.getTimeSeriesList()) {
+      final List<LabelValue> labelValuesWithPercentile =
+          new ArrayList<>(timeSeries.getLabelValues());
+      final io.opencensus.common.Timestamp timeSeriesTimestamp = timeSeries.getStartTimestamp();
+      for (io.opencensus.metrics.export.Point point : timeSeries.getPoints()) {
+        final io.opencensus.common.Timestamp pointTimestamp = point.getTimestamp();
+        point
+            .getValue()
+            .match(
+                Functions.<Void>returnNull(),
+                Functions.<Void>returnNull(),
+                Functions.<Void>returnNull(),
+                new Function<Summary, Void>() {
+                  @Override
+                  public Void apply(Summary summary) {
+                    Long count = summary.getCount();
+                    if (count != null) {
+                      createTimeSeries(
+                          timeSeries.getLabelValues(),
+                          Value.longValue(count),
+                          pointTimestamp,
+                          timeSeriesTimestamp,
+                          summaryCountTimeSeries);
+                    }
+                    Double sum = summary.getSum();
+                    if (sum != null) {
+                      createTimeSeries(
+                          timeSeries.getLabelValues(),
+                          Value.doubleValue(sum),
+                          pointTimestamp,
+                          timeSeriesTimestamp,
+                          summarySumTimeSeries);
+                    }
+                    Snapshot snapshot = summary.getSnapshot();
+                    for (ValueAtPercentile valueAtPercentile : snapshot.getValueAtPercentiles()) {
+                      labelValuesWithPercentile.add(
+                          LabelValue.create(valueAtPercentile.getPercentile() + ""));
+                      createTimeSeries(
+                          labelValuesWithPercentile,
+                          Value.doubleValue(valueAtPercentile.getValue()),
+                          pointTimestamp,
+                          null,
+                          percentileTimeSeries);
+                      labelValuesWithPercentile.remove(labelValuesWithPercentile.size() - 1);
+                    }
+                    return null;
+                  }
+                },
+                Functions.<Void>returnNull());
+      }
+    }
+
+    // Metric for summary->count.
+    if (summaryCountTimeSeries.size() > 0) {
+      addMetric(
+          metricsList,
+          createNewMetricDescriptor(
```

Comment:
This metric has alway the default unit "1". The sum and percentiles will have the metric unit.